library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -1242,16 +1233,11 @@
     <string name="encryption_import_room_keys">Import room keys</string>
     <string name="encryption_import_room_keys_summary">Import the keys from a local file</string>
     <string name="encryption_import_import">Import</string>
-<<<<<<< HEAD
     <string name="encryption_never_send_to_unverified_devices_title">Encrypt to verified devices only</string> <!-- Tchap: Use custom string -->
     <string name="encryption_never_send_to_unverified_devices_summary">Never send encrypted messages to unverified devices from this device.</string> <!-- Tchap: Use custom string -->
-=======
-    <string name="encryption_never_send_to_unverified_devices_title">Encrypt to verified sessions only</string>
-    <string name="encryption_never_send_to_unverified_devices_summary">Never send encrypted messages to unverified sessions from this session.</string>
     <string name="encryption_never_send_to_unverified_devices_in_room">Never send encrypted messages to unverified sessions in this room.</string>
     <string name="some_devices_will_not_be_able_to_decrypt">⚠ There are unverified devices in this room, they won’t be able to decrypt messages you send.</string>
     <string name="room_settings_global_block_unverified_info_text">🔒 You have enabled encrypt to verified sessions only for all rooms in Security Settings.</string>
```

Comment:
In Tchap, we replace `session` with `device`, please can you update these new strings (we should consider in the future if we want to switch back to `session`)

Reply:
I check in the current tchap version : sometimes it uses session, sometimes device.
How do we know which term to use ?
How can I identify new strings ?